bundles/org.openhab.binding.livisismarthome/src/main/java/org/openhab/binding/livisismarthome/internal/manager/DeviceStructureManager.java

```diff
@@ -44,8 +44,8 @@ public class DeviceStructureManager {
     private final Logger logger = LoggerFactory.getLogger(DeviceStructureManager.class);
 
     private final FullDeviceManager deviceManager;
-    private final Map<String, DeviceDTO> deviceMap;
-    private final Map<String, DeviceDTO> capabilityIdToDeviceMap;
+    private final Map<String, DeviceDTO> deviceMap = Collections.synchronizedMap(new HashMap<>());
```

Comment:
A reason to not use a ConcurrentHashMap like at the next line?

Reply:
I only moved it and did not analyse if it could be altered to a ` ConcurrentHashMap`. Now i did and i believe a lock on the entire object is not needed, a lock on the item would be enough. So changed it.